scripts/optimize_positions.py

```
@@ -0,0 +1,244 @@
+    
+    for camera in cameras:
+        # Extract camera parameters
+        initial_pos = np.array([
+            camera['position']['x'],
+            camera['position']['y'],
+            camera['position']['z']
+        ])
+        
+        # Define optimization bounds
+        bounds = [
+            (0, dimensions['length']),  # x
```

Ruff (0.8.2)

Undefined name `dimensions`

(F821)